src/OrchardCore.Modules/OrchardCore.ContentPreview/Views/ContentPreview.Button.cshtml

```diff
@@ -10,4 +10,4 @@
 <div id="previewContentItemId" style="display:none" data-value="@Model.ContentItem.ContentItemId"></div>
 <div id="previewContentItemVersionId" style="display:none" data-value="@Model.ContentItem.ContentItemVersionId"></div>
 
-<script asp-src="~/OrchardCore.ContentPreview/Scripts/contentpreview.edit.js" depends-on="admin" at="Foot"></script>
+<script asp-name="oc-contentpreview-edit" depends-on="admin" at="Foot"></script>
```

Comment:
the dependency should go in the resource manifest I imagine?

Reply:
done